validator/rpc/intercepter.go

```diff
@@ -17,11 +17,17 @@ import (
 // authentication from our API.
 var (
 	noAuthPaths = map[string]bool{
-		"/ethereum.validator.accounts.v2.Auth/Signup":             true,
-		"/ethereum.validator.accounts.v2.Auth/Login":              true,
-		"/ethereum.validator.accounts.v2.Auth/HasUsedWeb":         true,
-		"/ethereum.validator.accounts.v2.Wallet/HasWallet":        true,
-		"/ethereum.validator.accounts.v2.Wallet/GenerateMnemonic": true,
+		"/ethereum.validator.accounts.v2.Auth/Signup":                      true,
```

Comment:
In a future PR will invert this map to instead specify `authPaths`. There are few paths that actually require authentication after some discussion with a staker